lib/cybersource/ext/active_merchant/active_merchant.rb

```diff
@@ -70,6 +71,12 @@ def add_network_tokenization(xml, payment_method, options)
       end
 
       # Changes:
+      #  * http://apps.cybersource.com/library/documentation/dev_guides/Android_Pay_SO_API/html/wwhelp/wwhimpl/js/html/wwhelp.htm#href=ch_soAPI.html
+      #  * add paymentSolution tag to support Android Pay
+      def add_payment_solution(xml, payment_method)
+        xml.tag!("paymentSolution", "006") if (!payment_method.source.nil? && payment_method.source == :android_pay)
```

Comment:
Note that we [don't use](https://github.com/killbill/killbill-cybersource-plugin/blob/master/killbill-cybersource.gemspec#L38) the last ActiveMerchant version.

My question above was to check if we could use it in our case instead - it [doesn't look like it](https://github.com/activemerchant/active_merchant/blob/3120ee25370fbf0ebe606fa6a8f38eb0a7ebe6b4/lib/active_merchant/billing/network_tokenization_credit_card.rb).

We should probably introduce a new `source` plugin property - that way, it would be [forward compatible](https://github.com/killbill/killbill-plugin-framework-ruby/blob/d8691363d0f712a514727fff12c26b0ec54d04ca/lib/killbill/helpers/active_merchant/payment_plugin.rb#L544) with the latest AM implementation.